.github/workflows/node.js.yml

```diff
@@ -42,6 +42,16 @@ jobs:
                   verbose: true
                   file: ./coverage/coverage-final.json
                   flags: macos-unittests
+            - name: Code coverage (CodeWhisperer)
```

Comment:
what is the purpose of this? why isn't the existing codecov upload good enough? It is not scalable to add different codecov accounts for every toolkit feature.

Reply:
It's not adding a different codecov account but just doing a different upload for a new [flag](https://docs.codecov.com/docs/flags). See the following notes in the doc:
> It is important to note that for flagging to work, separate coverage reports must be uploaded and flagged per project in the mono repo
We are still running the build/test once. This action just does another upload for the flag tracker. There's only one codecov account involved.

Reply:
And now it shows another line of github check for `codecov/project/codewhisperer` showing code coverage

Reply:
> And now it shows another line of github check for `codecov/project/codewhisperer` showing code coverage

But we don't want CI to be constantly red. We intentionally removed this CI check in the past because it was noisy.

Reply:
We are planning to reach 75% in the next 1-2 weeks @crossrac is working on it.